packages/patternfly-3/patternfly-react-extensions/src/components/CatalogTileView/__mocks__/mockCatalogTileViewNoCategoriesExample.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { Icon } from 'patternfly-react';
+import { mockItems } from './mockItems';
+
+import { CatalogTile } from '../../CatalogTile';
+import { CatalogTileView } from '../index';
+
+const MockCatalogTileViewNoCategoriesExample = () => {
+  const getBadges = item => {
+    const badges = [];
+
+    if (item.certified) {
+      badges.push(
+        <CatalogTile.Badge key="certified" id="certified" title="Certified">
+          <Icon type="fa" name="cog" />
+        </CatalogTile.Badge>
+      );
+    }
+
+    if (item.approved) {
+      badges.push(
+        <CatalogTile.Badge key="certified" title="USDA Approved" id="approved">
```

Comment:
nit but i would order these the same as the certified badge - key/id/title
also is 'USDA' intended? would've thought simply Approved would be better